x-team/auto-ui
# Rename Tab to GenericTabs and Collapsible to GenericCollapsible

**Release Type:** *non breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-2838

## Description

Rename the new Tab and Collapsible to GenericTabs and GenericCollapsible respectively.

As part of the AC of the XP-2838 ticket a cleanup issue was created https://github.com/x-team/auto/issues/1820

## Checklist


## Steps to Test or Reproduce

1. run `npm start`
1. go to http://localhost:9001/?selectedKind=UI%20Components%2FGenericCollapsible
1. go to http://localhost:9001/?selectedKind=UI%20Components%2FGenericTabs
1. make sure to check the `Filters` demo http://localhost:9001/?selectedKind=UI%20Components%2FFilters as well as it's one of the components that makes usage of both renamed components

## Impacted Areas in Application

- `GenericCollapsible`
- `GenericTabs`

## Screenshots

_Checkout the Storybook stories for the renamed components and others that makes usage of them_

<img width="605" alt="screen shot 2019-02-18 at 15 18 29" src="https://user-images.githubusercontent.com/131859/52969805-769fc480-3390-11e9-90d5-9389b01da0e5.png">
